.github/workflows/nightly.yml

```diff
@@ -16,16 +16,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image_tag:
-          - "latest"
-          - "3.8"
-          - "3.7"
+        include:
+          - python: 3.6
+            image_tag: latest
+          - python: 3.7
+            image_tag: 3.8
+          - python: 3.8
+            image_tag: 3.7
```

Comment:
```suggestion
        image:
          - "latest"
          - "3.8"
          - "3.7"
        python:
          - "3.6"
          - "3.7"
          - "3.8"
```

Reply:
That would triple the number of tests run. I'd rather not do it.